src/main/resources/db/migration/nomis/ddl/V1_206__GL_TRANSACTIONS.sql

```diff
@@ -8,7 +8,7 @@ CREATE TABLE GL_TRANSACTIONS (
     TXN_TYPE VARCHAR2(6) NOT NULL,
     TXN_POST_USAGE VARCHAR2(12) NOT NULL,
     CASELOAD_ID VARCHAR2(6) NOT NULL,
-    OFFENDER_ID NUMBER(10,0) NOT NULL,
+    OFFENDER_ID NUMBER(10,0),
```

Comment:
Accidentally added this constraint that doesn't exist in production - can set offender_id to null (on instruction from Syscon)